Sources/_StringProcessing/_CharacterClassModel.swift

```diff
@@ -294,7 +298,17 @@ extension _CharacterClassModel: CustomStringConvertible {
 }
 
 extension _CharacterClassModel {
-  public func makeAST() -> AST.Node? {
+  public func makeDSLTreeCharacterClass() -> DSLTree.CustomCharacterClass? {
```

Comment:
Why is this public? Should it be SPI?

Reply:
`_CharacterClassModel` is SPI, so this is as well — I should probably at least add a comment so it isn't jarring to see this in an extension.